src/main.rs

```diff
@@ -91,11 +92,84 @@ struct ExecutionPayload {
     block_hash: String,
 }
 
-#[derive(Debug, serde::Serialize)]
-struct BlockSummary {
-    slot: String,
-    block_number: String,
-    block_hash: String,
+// #[derive(Debug, serde::Serialize)]
```

Comment:
Why is this commented?

Reply:
It's unused. Seems it's been like that for a while. I will remove it.